aten/src/ATen/native/native_functions.yaml

```diff
@@ -10254,6 +10253,12 @@
   dispatch:
     CPU, CUDA: linalg_lstsq_out
 
+- func: linalg_matrix_exp(Tensor self) -> Tensor
```

Comment:
Adding a new operator should be fine. There were a break last time, because some users are still using old selective build method, which is not guarded by CI. This kind of update should be captured now with internal CI and the operator list should be updated in the internal diff. cc @dhruvbird to confirm.

Reply:
OK. Confirmed that the old way of selective build is still being used. So removing an operator could break some existing users. We are trying to prioritize the deprecation of old selective build.